libraries/tools/kotlin-gradle-plugin/src/main/kotlin/org/jetbrains/kotlin/gradle/tasks/CompilerPluginOptions.kt

```diff
@@ -22,8 +22,8 @@ class CompilerPluginOptions {
     internal val subpluginOptionsByPluginId =
         mutableMapOf<String, MutableList<SubpluginOption>>()
 
-    val arguments: List<String>
-        get() = subpluginOptionsByPluginId.flatMap { (pluginId, subplubinOptions) ->
+    fun arguments(filteredPluginIds: Set<String> = emptySet()): List<String> =
```

Comment:
filtered usually means included, maybe `excludedPluginIds`?

Reply:
Removed in the latest commit.